brion/plugin/compartmentReportHDF5.cpp

```diff
@@ -260,6 +307,48 @@ bool CompartmentReportHDF5::writeFrame(const uint32_t gid, const float* values,
     return true;
 }
 
+bool CompartmentReportHDF5::writeFrame(const GIDSet& gids, const float* values,
+                                       const size_ts& sizes,
+                                       const double timestamp)
+{
+    if (gids.empty())
+        return true;
```

Comment:
is more like 'false' to me intuitively

Reply:
Not to me. If you ask me to read nothing, I will succeed reading nothing. Why should it be a failure?